inc/hclib-async.hpp

```diff
@@ -100,19 +99,19 @@ struct LambdaFutureWrapper<T, void> {
     }
 };
 
-/*
- * Yes, the name "async_at_hpt" sounds weird
- * but using this name to differentiate with the inter-node
- * "asyncAt" in HabaneroUPC++. Feel free to give a better
- * name to async_at_hpt.
- */
 template <typename T>
 inline void async(T &&lambda) {
     MARK_OVH(current_ws()->id);
     typedef typename std::remove_reference<T>::type U;
     hclib_async(lambda_wrapper<U>, new U(lambda), nullptr, nullptr, nullptr, 0);
 }
 
+/*
```

Comment:
Sure, I think my concern is now that HUPC++ is ripped out of the code base this comment/rationale doesn't make any sense anymore because it lacks context. I would be okay with renaming this API as part of this commit (e.g. async_at_place), but I also don't really understand why we need comments talking about how much someone in the past subjectively disliked a particular API name. It's not great, but it's not bad enough that anyone has felt really motivated to change it in the last year.